WebRoot/js/zimbraMail/briefcase/view/ZmPreviewPaneView.js

```diff
@@ -64,7 +64,21 @@ function() {
 
 ZmPreviewPaneView.SASH_THRESHOLD = 5;
 ZmPreviewPaneView._TAG_IMG = "TI";
+ZmPreviewPaneView.REMOVE_FILE_BUTTON_ID = "ZmPreviewPaneViewButton";
 
+ZmPreviewPaneView.getHTML = function (){
+    var html = [
+        "<table>",
+					"<tr>",
+						"<td>",ZmMsg.sharedFileAccess,"</td>",
+					"</tr>",
+                    "<tr>",
+                    "<td><button id='",ZmPreviewPaneView.REMOVE_FILE_BUTTON_ID,"'style='color:blue;background:none;border:none;margin:0;padding:0;cursor:pointer;font-size:14px;'>",ZmMsg.sharedFileRemove,"</button></td>",
```

Comment:
can we move this to templates? so zimlets can easily override HTML structure

Reply:
Moved the html to templates file. But the external css class could not be applied. Hence keeping it as inline style.

Reply:
it's important to move styling to css files

Reply:
Moved the styling to css file.